app/javascript/store/index.js

```diff
@@ -5,6 +5,7 @@ import profile from './modules/profile';
 import scores from './modules/scores';
 import statistics from './modules/statistics';
 import admin from './modules/admin';
+import froggo_team from './modules/froggo_team';
```

Comment:
Identifier 'froggo_team' is not in camel case    camelcase

Reply:
lo mismo acá :(